### PR TITLE
Feature/#31 data signal visualization from the node server

### PR DESCRIPTION
**Pull request by PhlppKrmr**
Worked on this issue with @jandegen & @motschel123

### CODE REVIEW

**Review by motschel123 — APPROVED** [review]
LGTM

**Comment by rabbit-zero** [comment]
looks fine

**Review by rabbit-zero — APPROVED** [review]

File: Apps/generator/src/main.rs [hunk 1]
```diff
@@ -26,29 +40,118 @@ async fn main() {
...
+        //}
+
+        //let bytearray: [u8; 80] = data.into_shape(80).unwrap().as_slice().expect("oops").try_into().expect("ooops");
 
         // send packet
-        match socket.send(&data) {
-            Ok(n) => println!("sent {n} bytes: {:?}", &data[..n]),
-            Err(e) => println!("failed sending: {e:?}."),
-        }
+        socket.send(&bytes);
```
Comment (jenswaechtler) [inline Apps/generator/src/main.rs:99]: Unused result

File: Apps/backend/package.json [hunk 2]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "backend",
-  "type":"module",
+  "type": "module",
```
Comment (jenswaechtler) [inline Apps/backend/package.json:3]: I find it very difficult to do a review when there are a lot of linter improvements in between. This should actually be in the scope of an extra pull request.

**Review by jenswaechtler — CHANGES_REQUESTED** [review]

**Comment by leon-juenemann** [comment]
Build upon this a bit and created a new PR as I didn't want to clutter this up. It changes quite a lot.
https://github.com/amosproj/amos2022ws03-software-oscilloscope/pull/58

**Review by jandegen — APPROVED** [review]

**Pull request merged**